project.clj

```diff
@@ -136,4 +137,5 @@
                               :jar-name "reset-password.jar"
                               ;; Exclude everything except for reset-password specific code in the created jar
                               :jar-exclusions [#"^(?!metabase/reset_password).*$"]
-                              :target-path "reset-password-artifacts/%s"}}) ; different than ./target because otherwise lein uberjar will delete our artifacts and vice versa
+                              :target-path "reset-password-artifacts/%s"} ; different than ./target because otherwise lein uberjar will delete our artifacts and vice versa
+             :h2-shell {:main ^:skip-aot org.h2.tools.Shell}})
```

Comment:
I don't think `^:skip-aot` should be necessary here. Check out this Stack Overflow it does a good job of expaining its purpose http://stackoverflow.com/questions/11174459/reason-for-skipping-aot